### PR TITLE
Create 22-jessehouwing.md

### PR DESCRIPTION
**Pull request by jessehouwing**

### CODE REVIEW

File: docs/_slides/22-jessehouwing.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Jesse!
```
Comment (kaufm) [inline docs/_slides/22-jessehouwing.md:1]: You are missing a lot of things here

```suggestion
---
layout: slide
title: "Welcome to our slide deck!"
---

Jesse

![dinotocat](https://octodex.github.com/images/dinotocat.png)
{: .center}
```

Comment (jessehouwing) [reply docs/_slides/22-jessehouwing.md]: Done!

File: docs/_slides/22-jessehouwing.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: slide
+title: "Jesse Houwing"
+---
+
+# Jesse Houwing
+
```
Comment (kaufm) [inline docs/_slides/22-jessehouwing.md:7]:
```suggestion
layout: slide
title: "Jesse Houwing"
---

# Jesse Houwing

```

File: docs/_slides/22-jessehouwing.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+layout: slide
+title: "Jesse Houwing"
+---
+
+# Jesse Houwing
```
Comment (kaufm) [inline docs/_slides/22-jessehouwing.md:6]: :+1:

File: docs/_slides/22-jessehouwing.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+layout: slide
+title: "Jesse Houwing"
```
Comment (kaufm) [inline docs/_slides/22-jessehouwing.md:3]: 👍

**Review by kaufm — APPROVED** [review]
🏅 Looks good

**Pull request merged**